WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -2,6 +2,9 @@
 
 class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
+  CERTIFICATE_PATH = Rails.root.to_s + "/../secrets/" + ENVied.CERTIFICATE_FILENAME
```

Comment:
How about instead of adding a new environment variable, we use the existing `ENVied.ROOT_URL`? You can see us using that to set up actionmailer [here](https://github.com/jfly/worldcubeassociation.org/blob/20222c80648de5f424385946fdf89d7a97cd53c9/WcaOnRails/config/environments/production.rb#L65-L70).